.travis.yml

```diff
@@ -203,7 +203,19 @@ script:
         make check-integration || exit 1
         make check-themes || exit 1
         do_codecov functionaltests
-        do_codecov_gcov c_code
+
+        # Run gcov.  Upload for codecov directly, gets done later for coveralls.
+        # NOTE: files for the main target have the build dir as prefix, but
+        #       otherwise it is the source directory?!
```

Comment:
`lgi-check` is created via `add_executable(lgi-check build-utils/lgi-check.c)`, i.e. a source-relative path. Instead, for the awesome executable we use absolute paths to the pre-processed `.c` files in the build dir (so relative to the source dir this means: there is an additional `build/` prefix).

Also: Thanks :-)